packages/react-components/src/components/FileDownloadErrorBar.tsx

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
@prprabhu-ms We have an ErrorBar component in already in our components.
Do you think we can re-use it for this?

Reply:
I thought about that a bit. I don't think that this is a good fit for `ErrorBar`.

`ErrorBar` does two things: (1) Holds business logic for how to interpret ACS service errors (via the selector) (2) Provide localized strings for shown errors.

In this case, we want only (2), but (1) is fundamental to the purpose of `ErrorBar`.

That said, see my comment above. I think we should use a `MessageBar` directly in `ChatComposite` with a localized string instead of piping error messages into `MessageThread`. (So this component will move to `react-composites`)

Reply:
Done